### DISCUSSION TITLE
Orca Tool Fix Model !

### DISCUSSION THREAD

**Discussion by beemergb** [OrcaSlicer/OrcaSlicer#4189] [General]
After writing a sketch in Fusion 360 and try to cut etc I always got ,"Inconsistent Graph Vertices" !! Drove me nuts !! Often !! So I saw in the tools in Orca the "Fix Model" Tool sent the STL from Fusion 360 to Orca opend it and applied "Fix Model Tool" and BAM !! I  transferd the now Fixed Model STl and it works a charm ! Magic , at last ! One Happy Camper

**Comment by aikiharp** [comment]
Are you on Windows? I haven't seen it in a while, but on a Mac Bambu/Orca/Prusa (and other derivatives, like Galaxy) would sometimes say the model needs fixing, but it would say in the same dialog that the model fixing function was only on Windows. Oh, ok--thanks! Regardless, my prints are 1000% better with Orca than with any other slicer. And it just keeps getting better. I love it.
